actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -98,6 +100,13 @@ def _filtered_referrer # :nodoc:
       #   the link. The drivers each provide mechanisms for listening for the
       #   completion of the Ajax request and performing JavaScript operations once
       #   they're complete
+      # * <tt>opener: true</tt> - This option prevents adding <tt>rel="noopener"</tt>
+      #   to an HTML output by default. <tt>rel="noopener"</tt> will be added to
+      #   a link by default, if <tt>target</tt> is <tt>_blank</tt> or any other
+      #   string except an empty string, <tt>_parent</tt>, <tt>_self</tt>, <tt>_top</tt>
+      #   and <tt>href</tt> is a url with <tt>http://</tt> or <tt>https://</tt>.
+      #   <tt>rel="noopener"</tt> nullifies <tt>window.opener</tt> (supported by selected browsers)
+      #   and prevents possible phishing attacks.
```

Comment:
Think we can go a bit more vinegary here with "noopener: false" to override. That also echoes the "noopener" language that everyone sees in security posts and such.

Reply:
Ah great news, if you need any help let me know @michald I would love to collaborate on this feature!